source/manual/ask-for-help.html.md

```diff
@@ -40,7 +39,3 @@ If you and your colleagues can’t resolve a technical issue, problem or questio
 1. The Technical Lead on the team
 2. The Lead Developer on the programme
 3. The Lead Architect
-
-If Technical 2nd Line instructs you to escalate something to GOV.UK Replatforming, raise a ticket on Zendesk and assign it to the `3rd Line--GDS Reliability Engineering` queue. You should also raise a ticket if the issue is related to an ongoing incident for tracking purposes, but you can speak to the team directly to get it more immediate attention.
-
-If you speak to GOV.UK Replatforming about a process only they know about, they will work with you to document the process for all of GOV.UK.
```

Comment:
I guess this being deleted means that all of the docs are up to date?

Reply:
Deleted because it was inaccurate (e.g. that Zendesk group is no longer in use) and it didn't make sense (i.e. if 2nd-line wanted something from Replatforming they'd just ask Replatforming; they wouldn't ask someone else to ask Replatforming!)

Reply:
I was talking about the last sentence:

>If you speak to GOV.UK Replatforming about a process only they know about, they will work with you to document the process for all of GOV.UK.

I think that should be kept?

Reply:
I don't think it's a particularly helpful — it's just boils down to "document stuff", which isn't always even the right choice.